_includes/pbc/all/install-features/202400.0/install-the-order-management-shipment-feature.md

```diff
@@ -0,0 +1,109 @@
+
+
+This document describes how to integrate the [Order Management](/docs/scos/user/features/{{page.version}}/order-management-feature-overview/order-management-feature-overview.html) + [Shipment](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/shipment-feature-overview.html) features into a Spryker project.
+
+{% info_block errorBox %}
+
+The following features integration guide expects the basic feature to be in place.
+
+The feature integration guide adds the following functionality:
+
+* [Order Management](/docs/scos/user/features/{{page.version}}/order-management-feature-overview/order-management-feature-overview.html)
+* [Shipment](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/shipment-feature-overview.html)
+
+{% endinfo_block %}
+
+## Install feature core
+
+Follow the steps below to install the Order Management + Shipment feature.
+
+### Prerequisites
+
+To start feature integration, integrate the required features:
+
+| NAME             | VERSION          | INTEGRATION GUIDE                                                                                                                                                                   |
+|------------------|------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Order Management | {{page.version}} | [Order Management feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/order-management-feature-integration.html)                                        |
+| Shipment         | {{page.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/unified-commerce/enhanced-click-and-collect/install-and-upgrade/install-the-shipment-feature.html) |
+
+
+## 1) Install the required modules using Composer
+
+```bash
+composer require spryker/sales-shipment-type:"^0.1.0" --update-with-dependencies
```

Comment:
It's a part of shipment feature IG.

Reply:
We discussed with @abitskil that we will create a separate IG for this. 
I don't mind moving it, but we need to discuss it first

Reply:
Let's add this to the shipment IG.

Reply:
Moved to main IG